sec_certs/helpers.py

```diff
@@ -224,6 +224,14 @@ def extract_pdf_metadata(filepath: Path):
     return constants.RETURNCODE_OK, metadata
 
 
+def to_utc(dt):
+    if dt.utcoffset() is None:
+        return dt
+    dt -= dt.utcoffset()
+    dt = dt.replace(tzinfo=None)
+    return dt
```

Comment:
Please, can we have more descriptive variable name and typehints?

Reply:
I can do a typehint but a better variable name I am not sure. It is a `datetime` but that's a class I don't want to overwrite.